### PR TITLE
Remade dates fields in trip/route

### PR DESCRIPTION
**Pull request by AndriiChepeliuk**

### CODE REVIEW

**Review by marynakernychna — APPROVED** [review]
Add space to the right of the date

**Review by marynakernychna — APPROVED** [review]

File: src/components/routes/route/index.jsx [hunk 1]
```diff
@@ -71,12 +71,7 @@ function UserRoute(props) {
                                 <div className="dates">
                                     <div className="date">
                                         <FiCalendar size={DEFAULT_ICON_SIZE} />
-                                        {moment(props.data.startDate).format('LLL') + " "}
-                                    </div>
-
-                                    <div className="date">
-                                        <AiOutlineArrowRight size={DEFAULT_ICON_SIZE} />
-                                        {moment(props.data.expirationDate).format('LLL')}
+                                        {moment(props.data.departureDate).format('LLL') + " "}
```
Comment (Grasmus) [inline src/components/routes/route/index.jsx]: Please, rewrite it with 24h format

File: src/components/myRoutes/myRoute/index.jsx [hunk 2]
```diff
@@ -64,12 +64,7 @@ function MyRoute(props) {
                                 <div className="dates">
                                     <div className="date">
                                         <FiCalendar size={DEFAULT_ICON_SIZE} />
-                                        {moment(props.data.startDate).format('LLL') + " "}
-                                    </div>
-
-                                    <div className="date">
-                                        <AiOutlineArrowRight size={DEFAULT_ICON_SIZE} />
-                                        {moment(props.data.expirationDate).format('LLL')}
+                                        {moment(props.data.departureDate).format('LLL') + " "}
```
Comment (Grasmus) [inline src/components/myRoutes/myRoute/index.jsx]: Please, rewrite it with 24h format

File: src/assets/scss/assets/pages/_routes.scss [hunk 3]
```diff
@@ -111,18 +131,30 @@
...
-            } 
+            }
         }
     }
+
     .ant-card.routeCard {
         width: 100%;
     }
-}
+}
```
Comment (Grasmus) [inline src/assets/scss/assets/pages/_routes.scss]: Add empty line in the eof

**Review by Grasmus — APPROVED** [review]

**Review by marynakernychna — APPROVED** [review]

**Pull request merged**